### DISCUSSION TITLE
Get all detected devices

### DISCUSSION THREAD

**Discussion by miczlo** [mKeRix/room-assistant#985] [Q&A]
I'm just starting with room-assistant so apologies if that's trivial, but is there a way to get a list of all detected MAC addresses over MQTT?
In terms of my use case - I'd like to use this to detect new unknown devices and track their appearance in time. 
I had a burglar in my garden some time ago, had a mask and gloves but pretty sure he had his phone with him and it would be nice to have his MAC address in case he ever comes around.

**Comment by luckygreen** [comment]
Bump.

Other than for the presence detection of individuals that are living at the house, who perhaps could be asked for the MAC addresses of their Bluetooth devices, room-assistant appears to be of rather limited utility absent an associated scanner that logs all the BT/BTE MAC addresses it sees, thereby allowing (manual) associations of names of guests with the MAC addresses of their devices. 

How are others performing scans of the BT/BTE RF environment to enable populating the list of MAC addresses for room-assistant to track? Is here a particular device or devices that are popular for this purpose? Thanks!
